### ISSUE TITLE
Issue with k8s.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/

### ISSUE THREAD

**Issue by manixybion** [kubernetes/website#31075]
https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/

I tried to use pv in k8s lab, I used the same doc given by k8s, It didn't work, 

When I try to curl the localhost after exec into the pod, I got 403, 
Also the index.html is not mounted into the nginx root dir

AM i missing anything..?

**Comment by sftim** [comment]
/kind support

At this stage it's not clear if the document needs fixing, or if it's more that the task is hard to follow accurately because it's a difficult thing to learn.

@manixybion would you be willing to share more details about what you tried, what choices you made, the cluster you used for this, and what command(s) you ran to see the problem?

**Comment by manixybion** [comment]
Yes, I think im missing something
